src/pages/channels/voice/VoiceHeader.tsx

```diff
@@ -85,13 +87,18 @@ export default observer(({ id }: Props) => {
 
     const client = useClient();
     const self = client.users.get(client.user!._id);
+    const state = useApplicationState();
 
     const keys = Array.from(voiceState.participants.keys());
     const users = useMemo(() => {
         return keys.map((key) => client.users.get(key));
         // eslint-disable-next-line
     }, [keys]);
 
+    if (voiceState.connecting) {
+        state.settings.sounds.playSound("call_join");
+    }
```

Comment:
Any re-render would cause the sound to play more than once here, I think this should be moved somewhere else.